config/toolchain/clang.cmake

```diff
@@ -1,15 +1,15 @@
 # Uncomment the following to use cross-compiling
 #set(CMAKE_SYSTEM_NAME Linux)
 
-set(CMAKE_COMPILER_VENDOR "clang")
```

Comment:
This caused the compiler checks to be skipped. Left code in case a change is needed after extensive testing.

Reply:
I will revert this because the reason this change was based on was incorrect.